src/localFunction.js

```diff
@@ -127,9 +127,22 @@ LocalFunction.prototype._requestCallBack = function(err, response, body) {
     return console.warn("\nERROR SENDING REQUEST: " + err);
   }
   var status = response ? response.statusCode + ", " : "";
-  return console.log(
-    "\nRESPONSE RECEIVED FROM FUNCTION: " + status + JSON.stringify(body, null, 2)
-  );
+
+  // If the body is a string we want to check if we can parse it as JSON
+  // and pretty-print it. We can't blindly stringify because stringifying
+  // a string results in some ugly escaping.
+  var bodyString = body;
+  if (typeof body === "string" || body instanceof string) {
```

Comment:
why both `typeof` and `instanceof` check?

Reply:
I dunno, the internet told me that was safest and I believe that JavaScript will always do the worst possible thing.